contracts/MasterChef.sol

```diff
@@ -0,0 +1,294 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.7.0;
+
+// import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
+import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+// import "@openzeppelin/contracts/math/SafeMath.sol";
+// import "./library/TransferHelper.sol";
```

Comment:
Lets remove remove the commented imports.

Reply:
Done